appinventor/components/src/com/google/appinventor/components/runtime/Web.java

```diff
@@ -887,8 +956,7 @@ private void performRequest(final CapturedProperties webProps, byte[] postData,
         processResponseCookies(connection);
 
         if (saveResponse) {
-          final String path = saveResponseContent(connection, webProps.responseFileName,
-              responseType);
+          final String path = saveResponseContent(connection, webProps.responseFileName, responseType);
```

Comment:
I think IntelliJ reformatted this line on you. Revert this change.

Reply:
Fixed this